### PR TITLE
LPD-34244 

### PR DESCRIPTION
**Pull request by joshuacords**
**Story**: https://liferay.atlassian.net/browse/LPD-28678
**Subtask**: https://liferay.atlassian.net/browse/LPD-34244

AC on the Story. Tests will be added with the other subtask.

**Author:** @joshuacords

### CODE REVIEW

**Comment by joshuacords** [comment]
ci:test:search

**Comment by joshuacords** [comment]
ci:test:sf

**Review by gustavolimav — CHANGES_REQUESTED** [review]
Code in general Looks Good. Just one question, I tested locally and it is passing the AC

File: modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/parameter/contributor/UserSXPParameterContributor.java [hunk 1]
```diff
@@ -155,10 +156,17 @@ public String getSXPParameterCategoryNameKey() {
 				new SXPParameterContributorDefinition(
 					StringSXPParameter.class, "email-domain",
 					"user.email_domain"),
+				new SXPParameterContributorDefinition(
+					StringArraySXPParameter.class, "external-reference-code",
```
[the remark this answers is not in the document]
Comment (joshuacords) [reply modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/parameter/contributor/UserSXPParameterContributor.java:160]: Yeah, good catch thank you!

**Comment by joshuacords** [comment]
ci:forward

**Pull request closed without merging**